.travis.yml

```diff
@@ -4,7 +4,7 @@ language: php
 
 branches:
   except:
-    - /^release-.*$/
+    - /^release-\d+\.\d+\.\d+.*$/
     - /^ghgfk-.*$/
```

Comment:
IIRC, these were from a tool @Maks3w was using for merge automation.

The `release-` regex is so that tests are not run for tagged releases; the assumption is that tests *have already run*, and this prevents release tags from taking up space in the Travis queue. The problem is that the previous regex would have made it impossible to have a long-lived `release-X.Y` branch (which we've created a few times to backport changes). The new regex ensures that only a tagged release version will lead to the exclusion.